### ISSUE TITLE
Version 2 Plan - Features and Enhancements Roadmap

### ISSUE THREAD

**Issue by visto9259** [LM-Commons/LmcRbac#16]
## Version 2 Plan - Features and Enhancements Roadmap
This is to document the enhancements, features and other changes to bring to LmcRbac in a forthcoming version 2.

### Rationale
LmcRbac v1 is a port of ZF-Commons/ZfcRbac v3 to Laminas.  ZfcRbac v3 was developed several years ago and was based on Zend Framework v2 and v3.  ZfcRbac v3 was created to be a lighter weight library of ZfcRbac v2 that had MVC components.

The port of ZfcRbac to Laminas has not brought significant improvements or new features to the package.  

LmcRbac has been installed about 2K times according to Packagist which is an indication that the package has some usefulness.  We are looking for users of the package to provide inputs into the development roadmap for Version 2. 

### Process
- Comment on this issue if you want to add an enhancement or a feature to be included in Version 2.  
- Issues will be created for enhancements/features selected for inclusion in V2 and will be added to the development Backlog and will be labeled as `v2`
- Development toward V2 will be done on the `2.x` branch.
- A GitHub project for LmcRbac was created and issues to be added to the project so that progress can be managed
- Issues will be assigned to contributors

## Selected Enhancements
- [x] #48 
- [x] Have LmcRbacMvc use LmcRbac for Rbac, Roles, services (#49)
- [x] #53
- [x]  #55
- [x] #61 
- [x] #62 
- [x] #65 

## Proposed enhancements
- (**DROPPED**) Move Doctrine Support to a separate library (#12)

**Comment by visto9259** [comment]
@basz @prolic 
Are you guys still involved with this repo?  Are you still using it?
If not, that's ok.  I will take over as I want to bring changes to it.

**Comment by basz** [comment]
Yeah I still use this package in a single project but doing no active development on it anymore. You are very much welcomed to take over. 👍

**Comment by visto9259** [comment]
Adding #48 to list of proposed enhancements

**Comment by visto9259** [comment]
Adding #53 to the list of enhancements

**Comment by visto9259** [comment]
all items have been completed
